HW-0.Rmd

```diff
@@ -1,14 +1,14 @@
 ---
-title: "Testing"
-author: "WRITE YOUR NAME HERE"
-date: "February 3, 2016"
-output: html_document
+title: "HW-0"
+author: "Aminata Deme"
+date: "February 24, 2016"
+output: http://rpubs.com/AminataDeme6/155533
 ---
 
 Your tasks!
 
 * Change the name of the author above to yours
 * Change the date
-* Replace the NHL URL with the URL for the document you just posted to Rpubs **here**: [My RPubs Document](https://www.nhl.com/)
+* Replace the NHL URL with the URL for the document you just posted to Rpubs **http://rpubs.com/AminataDeme6/155533**: [My RPubs Document](https://www.nhl.com/)
 * Click Knit HTML
```

Comment:
Close enough. You had to insert the link so that the code looked like:

```
[My RPubs Document](http://rpubs.com/AminataDeme6/155533)
```

That way when you Knit HTML the document, you have a blue link that says [My RPubs Document](http://rpubs.com/AminataDeme6/155533)